sources/Likutei Halachot/fill_in_english_titles.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+import csv
+import urllib2
+from get_section_titles import get_he_section_title_array
+from fuzzywuzzy import fuzz
+from fuzzywuzzy import process
+
+def match_titles(he_sections, tur_table_csv_url):
+    url = 'http://winterolympicsmedals.com/medals.csv'
+    response = urllib2.urlopen(tur_table_csv_url)
+    reader = csv.reader(response)
+    tur_table = list(reader)
+    
+    #make corresponding list to use for fuzzywuzzy
+    tur_he_title_list = []
+    for row in tur_table:
+        tur_he_title_list.append(row[0])
+    
+    tur_he_title_list.pop(0)
+    
+    #make array to reutrn
+    he_and_en_title_list = []
+    
+    #match titles
+    for section in he_sections:
+        result = get_highest(section, tur_he_title_list)
+        if result[1] >= 70:
+            he_and_en_title_list.append([u"%s" % section, tur_table[tur_he_title_list.index(result[0])+1][2]])
+        else:
+            he_and_en_title_list.append([section, ""])
+    
+    return he_and_en_title_list;
+
+def get_highest(term, input_list):
+    score=-1
```

Comment:
Not critical, but it's a good practice to write floats with a decimal (i.e. -1.0 instead of 1)